From ca905ef39a1ecd7a995bce15c5b740820e07cba5 Mon Sep 17 00:00:00 2001
From: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Date: Fri, 21 Sep 2018 09:40:03 +0700
Subject: [PATCH 621/909] arm64: dts: r8a77990-ebisu-4d: Add lossy phandle to
 MMNGR driver node

commit e73fb3f74840668b1e75451a765ba1c22ffb8435 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds the lossy descompress area phandle to MMNGR
driver node and allow it to parse reserved memory for
lossy compression feature.

Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu-4d.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu-4d.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu-4d.dts
index 49017ce0d800..1c19504c672b 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu-4d.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu-4d.dts
@@ -25,6 +25,10 @@
 			reg = <0x00000000 0x54000000 0x0 0x03000000>;
 		};
 	};
+
+	mmngr {
+		memory-region = <&mmp_reserved>, <&lossy_decompress>;
+	};
 };
 
 &pciec0 {
-- 
2.17.1

